build_ngtf.py

```diff
@@ -432,7 +432,7 @@ def main():
     #-------------------------------
 
     # Component versions
-    ngraph_version = "d3453447883fe48e63126316d5da72fcbd47dded"
+    ngraph_version = "df9e23761f5f7a7a7d0134c8ae0c92a8bad2e961"
```

Comment:
This should be set to release tag i.e., `v0.15.0-rc.1` at this point. I will add this change.